packages/react/src/generators/application/files/common-vite/.browserslistrc__tmpl__

```diff
@@ -0,0 +1,16 @@
+# This file is used by:
```

Comment:
Same with browserlist, we don't need to generate is since Vite doesn't support it out of the box.